### ISSUE TITLE
glottocode to Jakarta Indonesian source

### ISSUE THREAD

**Issue by HedvigS** [glottolog/glottolog#876]
This reference:

- Sneddon, James Neil. 2006. Colloquial Jakartan Indonesian. (Pacific Linguistics, 581.) Canberra: Research School of Pacific and Asian Studies, Australian National University. xi+286pp.

Should probably be associated with this glottocode: cjin1234 (either instead of or in addition to what it now has)

**Comment by barlowrussell** [comment]
This raises another issue, though. Since there are currently no sources associated with "Colloquial Jakarta Indonesian", it's hard to know how Glottolog defines this lect. If this lect *is* associated with Sneddon (2006) (and I imagine that it would be), then maybe it should be nested within Standard Indonesian [indo1316] rather than within Betawi [beta1252]?

Sneddon (2006:5-6) writes: "A distinction needs to be made between CJI [Colloquial Jakartan Indonesian] and Jakarta Malay. While there has been little study of CJI, descriptions of Jakarta Malay, also called Betawi Malay, are available and there is a Jakarta Malay-Indonesian dictionary. Jakarta Malay is sometimes confused with CJI. However, although Jakartan Indonesian has been considerably influenced by Jakarta Malay it is nevertheless distinct from it. Jakarta Malay is the vernacular of the *anak Betawi* ‘children of Batavia’, inhabitants of ‘the old kampungs’ of Jakarta, whose families have usually lived in Jakarta for many generations. One writer has shown that the *anak Betawi*, who constitute an ever-dwindling percentage of the overall Jakartan population, shift to informal Jakartan Indonesian in the presence of strangers; it is an ‘in-group’ code and rarely heard by people from outside the Betawi communities. Another points out that young families in Jakarta who speak Betawi Malay shift to Jakartan Indonesian ‘when they rise socially or have aspirations in that direction’. This shows that the difference is clearly discernible and that CJI is associated with a higher social status. It is the everyday language of the great majority of Jakartans, including the elite and the relatively large educated middle-class."

**Comment by d97hah** [comment]
Hi! Sorry to be slow to react to this. Parental leave does that to
you.  Colloquial Jakarta Indonesian [cjin1234] is classified under
Betawi because people like Collins (2022) and Tadmor (2018) say that
Colloquial Jakarta Indonesian is "based on Betawi Malay" and
"Jakarta Indonesian has developed from Betawi".

The language in Sneddon (2006), to me, does not look like it is "based
on" or "developed from" the language described in e.g. Ikranagara
(1980). It rather looks like Sneddon (2006) is standard Indonesian
with some important verb prefix alternations and lexical differences,
some of which are shared with Ikranagara's (1980) language, e.g., the
negation and some pronouns. But most conspicuously Sneddon (2006)'s
language has final -a where Ikranagara 1980's language has a rule
making it -e (or even schwa), in e.g., ade, nye, punye, die
etc. Sneddon (2006) doesn't have a phonology section but such an
omnipresent difference from Standard Indonesian would have been
remarked upon if it was there, I would think.

Sneddon (2006) (and other authors) say there's a continuum of informal
varieties, and Sneddon (2006) says he describes the speech of educated
people. I assume them there then another CJI of uneducated people that
is derived from Betawi Malay that is what Collins/Tadmor etc are talking
about.

So to sum up, the current Glottolog seems correct enough. Sneddon
(2006) should be associated with [ind] rather than with [bew]. CJI can
apparently mean different things and its classification with Betawi is
possibly wrong but I can't assert a better alternative than to believe
Collins/Tadmor's words, at least until they explain in more detail
what they mena.

all the best, H


Pada tanggal Jum, 9 Sep 2022 pukul 11.27 barlowrussell <
***@***.***> menulis:

**Comment by barlowrussell** [comment]
Thanks for clarifying, Harald! That makes a lot of sense. Should there then also be a dialect "Colloquial Jakarta Indonesian" nested within "Standard Indonesian"? Sneddon (2006) is, I would think, the work most associated with the term "Colloquial Jakarta Indonesian", so maybe having a lect with that name listed (only) under Betawi might lead people (like me) to think that the language in Sneddon's work is being associated genealogically with Betawi.

**Comment by d97hah** [comment]
Maybe, but I suspect the Collins/Tadmor classification and/or wording is
not right. Wouk (1989)'s language ("Spoken Jakarta Indonesian")
is from "at least highschool graduates, all from the middle socioeconomic
class" and looks identical to that of Sneddon (2006), and similarly doesn't
contain a phonology. Both Wouk & Sneddon are careful not to say from where
their languages derive. I'll try to go to the bottom of Collins/Tadmor's
claims and then fix. all the best, H

Pada tanggal Jum, 7 Okt 2022 pukul 13.25 barlowrussell <
***@***.***> menulis:

**Comment by HedvigS** [comment]
Okay, forgive me if I've misunderstood something.

This is my reading: what Sneddon 2006 is describing is not something that should be under Betawi. We want either cjin1234 to be under indo1316 or for there to be a new dialect glottocode under indo1316 that refers to what Sneddon is describing.

**Comment by d97hah** [comment]
Ok I've just done a PR adding Acrolectal CJI under [ind] if [ind] is not
detailed enough a code for Sneddon 2006. all the best, H

Pada tanggal Sel, 10 Jan 2023 pukul 11.12 Hedvig Skirgård <
***@***.***> menulis:

**Comment by xrotwang** [comment]
See https://github.com/glottolog/glottolog/pull/922/files

**Comment by HedvigS** [comment]
Wait. I don't understand. 
Should cjin1234 really still be under beta1260 given the change in the commit Robert just linked to?

**Comment by xrotwang** [comment]
> This is my reading: what Sneddon 2006 is describing is not something that should be under Betawi. We want either cjin1234 to be under indo1316 or for there to be a new dialect glottocode under indo1316 that refers to what Sneddon is describing.

@d97hah chose option 2, and `acro1241` is that Glottocode.

**Comment by HedvigS** [comment]
Thank you for clarifying that Robert, it was just tricky to follow.

We will change the coding sheet assignment for us in GB then from cjin1234 to acro1241 for what Sneddon describes.
